lionblog-yeeun/src/components/Posts/index.jsx

```diff
@@ -10,9 +10,36 @@ export const SmallPost = ({ post }) => {
           </span>
         ))}
       </div>
-      <div>
-        {post.like_users.length > 0 && `❤️ ${post.like_users.length}`}
+      <div>{post.like_users.length > 0 && `❤️ ${post.like_users.length}`}</div>
+    </div>
+  );
+};
+
+export const BigPost = ({ post }) => {
+  const pressLike = () => {
```

Comment:
like를 누르는 이벤트가 발생했을 때 실행되는 함수네요! 이런 이벤트 핸들러 함수의 경우 보통 handle어쩌구...로 이름을 짓는 것이 일반적입니다!
```suggestion
  const handleLike = () => {
```